### PR TITLE
Add Run button if env var is present

### PR DESCRIPTION
**Pull request by thomasballinger**
Add run button if env variable `COMMUTER_EXECUTE_LINK` is present.

We had to thread this data through differently on the client and server side.

paired with @betatim and @rgbkrk  on this

### CODE REVIEW

**Comment by thomasballinger** [comment]
cc @cabhishek

**Comment by thomasballinger** [comment]
@cabhishek does the way we're building this path work for you? e.g. ENV_VAR + `/dirA/dirB/file.ipynb`

File: packages/commuter-frontend/pages/view.js [hunk 1]
```diff
@@ -42,10 +60,23 @@ class ViewPage extends React.Component<*> {
...
+    super(props);
+    if (props.serverConfig) {
+      this.state = { config: props.serverConfig };
+    } else {
+      const config = JSON.parse(
+        document.getElementById("serverConfig").textContent
+      );
+      this.state = { config };
+    }
+  }
```
Comment (rgbkrk) [inline packages/commuter-frontend/pages/view.js]: Ah yep that’s exactly what I was expecting.

**Comment by cabhishek** [comment]
@thomasballinger yes, ` ENV_VAR + /dirA/dirB/file.ipynb` works 👍 . Thanks

**Review by cabhishek — APPROVED** [review]

**Comment by rgbkrk** [comment]
I think if you rebase this one @thomasballinger we can merge it in and plow ahead.

**Comment by rgbkrk** [comment]
Thanks all!

**Pull request merged**